example/index.html

```diff
@@ -1,13 +1,15 @@
 <html>
   <head>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-    <title>NPM RUN FASTER</title>
+    <title>Working</title>
     <!-- <script src="/client.js"></script> -->
-    <script src="https://rawgit.com/ideaq/faster/master/lib/client.js"></script>
+    <script src="http://192.168.0.27:4242/client.js"></script>
```

Comment:
Reminder to move this back to the CDN in the final tidy-up after development :wink: